### PR TITLE
Allow pole-mounted transformers

### PR DESCRIPTION
**Pull request by Famlam**
Issue #1329 

Besides power=transformer for a standalone transformer, also power=pole + transformer=* is a valid transformer tagging. See https://wiki.openstreetmap.org/wiki/Tag:power=transformer and https://wiki.openstreetmap.org/wiki/Tag%3Atransformer%3Ddistribution

Fixes i.e. https://www.openstreetmap.org/node/1980809160, https://www.openstreetmap.org/node/3159270648, https://www.openstreetmap.org/node/3269583658 (examples for three different values of transformer=* on pole)

Verified all results on germany_schleswig_holstein, excludes 15 cases of the 74 detected errors. 
(Tested on the dev server, which is many times faster than my local pc, thanks!)

### CODE REVIEW

**Comment by frodrigo** [comment]
> (Tested on the dev server, which is many times faster than my local pc, thanks!)

So, good to know it helps you.

File: analysers/analyser_osmosis_powerline.py [hunk 1]
```diff
@@ -221,9 +221,12 @@
...
         power_line.nid = nodes.id AND
-        (
-            NOT nodes.tags?'power' OR
-            nodes.tags->'power' != 'transformer'
+        NOT (
+            nodes.tags?'power' AND
+            (
+                nodes.tags->'power' = 'transformer' OR
+                nodes.tags?'transformer' -- example: power=pole + transformer=*
+            )
```
[the remark this answers is not in the document]
Comment (Famlam) [reply analysers/analyser_osmosis_powerline.py:229]: @frodrigo Is `nodes.tags->'power' != 'transformer'` ever executed?

`nodes.tags?'power' OR nodes.tags->'power' != 'transformer'`
is for a node **with** power tag basically the same as
`TRUE OR -not executed-`
and for a node **without** power tag, it's the same as
`FALSE OR null != 'transformer'` => `FALSE OR FALSE`

Not sure if there's anything else I can change?

[the remark this answers is not in the document]
Comment (frodrigo) [reply analysers/analyser_osmosis_powerline.py:229]: Right, I fix my suggestion.

**Comment by frodrigo** [comment]
Good. Thank you.

**Comment by frodrigo** [comment]
Deployed.

**Comment by Famlam** [comment]
I think the corresponding issue (1329) can be closed now, right? Or do I need to adjust something?

**Pull request merged**